### PR TITLE
drafting a blog post about joining as an associate member

### PR DESCRIPTION
**Pull request by DuaneOBrien**
Adds a draft blog post describing the path to joining TODO as an Associate Member.

### CODE REVIEW

File: content/en/blog/becoming-a-todo-group-member-as-an-associate.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+The TODO Group is a Linux Foundation collaborative project that serves as a working group for open source program office members around the world. In order to join the TODO Group, an organization needs to also be a member of the Linux Foundation. Many of the members of the Linux Foundations are companies or corporations who pay annual membership fees and also are paying members of one of the many sub-foundations hosted by the LF. But what do you do if you represent a muncipality or academic institution who is interested in forming an open source programs office? 
+
+Fortuntely, the Linux Foundation also offers an Associate Membership for civic, academic, and non-profit institutions who wish to join the LF and the TODO Group. Joining as an Associate Member is a simple, free process that opens up access to the TODO Group for any qualifying institution.
+
+You can view the Associate Membership application in the TODO Group's [Governance](https://github.com/todogroup/governance) repository, which also contains the TODO Charter and Membership Agreement. If you wish to join as an Associate Member, please email info@todogroup.org for more information.
```
Comment (vmbrasseur) [inline content/en/blog/becoming-a-todo-group-member-as-an-associate.md:13]: This is unclear. Is it an application to become an Assoc. member of LF? Or, being an LF Assoc. member already, to join TODO Group? If the former, then we should kick this to LF. It's not our place to manage their associate membership process or to host the application. If the latter then we need to be more clear about the workflow: step 1: join LF as an associate member (and link to the page/process for that); step 2: after gaining LF assoc membership, send us $this_information to $that_email. (or whatever automated process Amye mentioned the other day)

File: content/en/blog/becoming-a-todo-group-member-as-an-associate.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Becoming a TODO Group Member as a Linux Foundation Associate Member"
+author: todogroup
+date: 2020-05-07
+---
+
+Open source program offices have become a best practice for companies that rely on open source technologies. As open source program offices have spread, there has been increased interest in applying this best practice in non-corporate contexts. Cities, universities, and other non-corporate organizations are starting to see the benefits of creating a center of competency for open source.
+
+The TODO Group is a Linux Foundation collaborative project that serves as a working group for open source program office members around the world. In order to join the TODO Group, an organization needs to also be a member of the Linux Foundation. Many of the members of the Linux Foundations are companies or corporations who pay annual membership fees and also are paying members of one of the many sub-foundations hosted by the LF. But what do you do if you represent a muncipality or academic institution who is interested in forming an open source programs office?
```
[the remark this answers is not in the document]
Comment (DuaneOBrien) [reply content/en/blog/becoming-a-todo-group-member-as-an-associate.md]: Integrated some of this and changed the language a bit here to address these comments.

File: content/en/blog/becoming-a-todo-group-member-as-an-associate.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+The TODO Group is a Linux Foundation collaborative project that serves as a working group for open source program office members around the world. In order to join the TODO Group, an organization needs to also be a member of the Linux Foundation. Many of the members of the Linux Foundations are companies or corporations who pay annual membership fees and also are paying members of one of the many sub-foundations hosted by the LF. But what do you do if you represent a muncipality or academic institution who is interested in forming an open source programs office? 
+
+Fortuntely, the Linux Foundation also offers an Associate Membership for civic, academic, and non-profit institutions who wish to join the LF and the TODO Group. Joining as an Associate Member is a simple, free process that opens up access to the TODO Group for any qualifying institution.
```
[the remark this answers is not in the document]
Comment (DuaneOBrien) [reply content/en/blog/becoming-a-todo-group-member-as-an-associate.md]: Added the definition of qualifying associate members and linked out to the LF's bylaws for further context.

File: content/en/blog/becoming-a-todo-group-member-as-an-associate.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Becoming a TODO Group Member as a Linux Foundation Associate Member"
```
[the remark this answers is not in the document]
Comment (DuaneOBrien) [reply content/en/blog/becoming-a-todo-group-member-as-an-associate.md:2]: Initially I agreed with you, but after going through the rest of the comments and tweaking the language, I think this is the right title. I'll expand on why below.

File: content/en/blog/becoming-a-todo-group-member-as-an-associate.md (hunk 1 above)
Comment (DuaneOBrien) [reply content/en/blog/becoming-a-todo-group-member-as-an-associate.md]: Maybe what we need here is a short, specific call-out for the steps. "If you want to join TODO, and you work in a non-profit, academic, or civic organization: 1 - Apply for associate membership in the LF; 2- Once associate membership has been approved, email info@todogroup.org"

To clarify, the goal of this specific blog post is to clarify the path to membership in TODO for orgs that would normally only have that path through Associate Membership. I drafted the blog post to respond to inquiries about how academic or municipal institutions with budding open source programs could join TODO. The intent of the blog post isn't to document this process for the LF, it's to better serve prospective members by having something documented that we can point them to. 

Ultimately that's why I think the title "Becoming a TODO Group Member as a Linux Foundation Associate Member" is closer to the mark than "Joining the TODO Group as a Linux Foundation Associate Member" - the latter reads to me more like "If you're already a LF Associate Member, here's how to join TODO" when what I'm trying to go for is "If you want to join TODO and you're not a company, here's the path"

File: content/en/blog/becoming-a-todo-group-member-as-an-associate.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
```
Comment (jeffmcaffer) [inline content/en/blog/becoming-a-todo-group-member-as-an-associate.md:2]: The suggestion below captures the problem the blog solves. 

```suggestion
title: "How non-profits, governments, and universities can join the TODO Group"
```

File: content/en/blog/becoming-a-todo-group-member-as-an-associate.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+The TODO Group is a Linux Foundation collaborative project that serves as a working group for open source program office members around the world. In order to join the TODO Group, an organization needs to also be a member of the Linux Foundation. Members of the Linux Foundation are typically companies or corporations that pay annual membership fees. But what do you do if you represent a muncipality or academic institution who is interested in forming an open source programs office? 
+
+Fortuntely, the Linux Foundation also offers an Associate Membership for civic, academic, and non-profit institutions that wish to become Linux Foundation members and join the TODO Group. Joining as an Associate Member is a simple, free process that opens up access to the TODO Group institution that meet the criteria for Associate Membership as spelled out in the [Linux Foundation's Bylaws.](https://www.linuxfoundation.org/bylaws/)
```
Comment (jeffmcaffer) [inline content/en/blog/becoming-a-todo-group-member-as-an-associate.md:11]:
```suggestion
Fortuntely, the Linux Foundation also offers an Associate Membership for civic, academic, and non-profit institutions that wish to become Linux Foundation members and join the TODO Group. Joining as an Associate Member is a simple, free process that opens up access to the TODO Group. The criteria for Associate Membership is spelled out in the [Linux Foundation's Bylaws.](https://www.linuxfoundation.org/bylaws/) as follows:
```

File: content/en/blog/becoming-a-todo-group-member-as-an-associate.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+
+> Associate Members shall be non-profit and government entities that engage in or support the production, manufacture, use, sale, or standardization of Linux or other open source-based technologies. An Associate Member shall have no voting rights, and shall meet such eligibility criteria and pay such fees (if any) as the Board of Directors may from time to time approve. An entity shall cease to be an Associate Member in the event of its resignation or expulsion from this corporation.```
+
+You can view the Linux Foundation Associate Membership application in the TODO Group's [Governance](https://github.com/todogroup/governance) repository, which also contains the TODO Charter and Membership Agreement. If you wish to join as an Associate Member, please email info@todogroup.org for more information.
```
Comment (jeffmcaffer) [inline content/en/blog/becoming-a-todo-group-member-as-an-associate.md:15]: Just curious, does the LF have the associate membership app somewhere on the web? Seems odd that we have a copy in our repo.

Comment (DuaneOBrien) [reply content/en/blog/becoming-a-todo-group-member-as-an-associate.md:15]: When I drafted the blog post, they did not. Checking it into our repo was my fix for that, so we could draft this guidance. Prior to that, when I asked where to find the application, the answer was "email and ask for it"

File: content/en/blog/becoming-a-todo-group-member-as-an-associate.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+
+The TODO Group is a Linux Foundation collaborative project that serves as a working group for open source program office members around the world. In order to join the TODO Group, an organization needs to also be a member of the Linux Foundation. Members of the Linux Foundation are typically companies or corporations that pay annual membership fees. But what do you do if you represent a muncipality or academic institution who is interested in forming an open source programs office? 
+
+Fortuntely, the Linux Foundation also offers an Associate Membership for civic, academic, and non-profit institutions that wish to become Linux Foundation members and join the TODO Group. Joining as an Associate Member is a simple, free process that opens up access to the TODO Group. The criteria for Associate Membership is spelled out in the [Linux Foundation's Bylaws.](https://www.linuxfoundation.org/bylaws/) as follows:
```
Comment (jeffmcaffer) [inline content/en/blog/becoming-a-todo-group-member-as-an-associate.md:11]:
```suggestion
Fortunately, the Linux Foundation also offers an Associate Membership for civic, academic, and non-profit institutions that wish to become Linux Foundation members and join the TODO Group. Joining as an Associate Member is a simple, free process that opens up access to the TODO Group. The criteria for Associate Membership are spelled out in the [Linux Foundation's Bylaws.](https://www.linuxfoundation.org/bylaws/) as follows:
```

File: content/en/blog/becoming-a-todo-group-member-as-an-associate.md (hunk 7 above)
Comment (jeffmcaffer) [reply content/en/blog/becoming-a-todo-group-member-as-an-associate.md:15]: Interesting. Thanks for the context.

**Comment by DuaneOBrien** [comment]
Is there anything currently blocking this or are we good to merge?

**Comment by jeffmcaffer** [comment]
:shipit:

**Pull request merged**